apps/docs/src/components/Layout.jsx

```diff
@@ -40,7 +52,6 @@ export const navigation = [
     title: 'Contributing',
     links: [
       { title: 'How to contribute', href: '/docs/how-to-contribute' },
-      { title: 'Architecture guide', href: '/docs/architecture-guide' },
```

Comment:
You need to update apps/docs/src/app/page.md too (since you moved /docs/architecture-guide to /docs/architecture)